### DISCUSSION TITLE
Trainning hifigan for FastPitch

### DISCUSSION THREAD

**Discussion by jonathandasilvasantos** [coqui-ai/TTS#3459] [General Q&A]
Hello everyone. I've been experimenting with coquiTTS on various models, including glowTTS, vits, FastPitch, and others...
I'm very pleased with the result of FastPitch as it is very fast, enabling reasonable CPU inference for my purposes. I've been training models in Brazilian Portuguese (pt-br).

However, unfortunately, I'm unable to train a hifigan vocoder to use with FastPitch. Using the recipe from ljspeech/hifigan, I can use the vocoder with glowTTS, but with FastPitch, I only hear noise.

When I use the pretrained model vocoder_models/en/ljspeech/hifigan_v2, the result is good, but I would like to improve it. Therefore, does anyone have any idea how I can fine-tune the vocoder_models/en/ljspeech/hifigan_v2 for FastPitch?

Or even a from scratch recipe to train hifigan that works with FastPitch?

Thank you very much!
Jonathan S. Santos

**Comment by enjoy301** [comment]
Same here

**Comment by Sub4t7** [comment]
I really tried but unfortunately I was unsuccessful. I would really like someone to train hifigan to support Brazilian Portuguese, I've been wanting this for a long time. If anyone knows how to do it, I can help with hardware (a gaming computer, alone it's not much but in a group we multiply the hardware), if we can get a group of people together, knowing exactly what and how it should be done, we can make hifigan work in pt-br. 
**It's already January 8, 2025 and so far we don't have support for pt-br**, if we don't come together and work on this, it hasn't come yet, it won't come any other way by the looks of it, when do we start?)
